src/main/java/com/duke/task/Todo.java

```diff
@@ -0,0 +1,26 @@
+package com.duke.task;
+
+/**
+ * Represents a Todo task.
+ */
+public class Todo extends Task {
+    /**
+     * Creates a Todo task with the description given.
+     *
+     * @param d description of the Todo task.
+     */
+    public Todo(String d) {
+        super(d);
+    }
+
+    @Override
+    public String toString() {
+        return "[T]" + super.toString();
+    }
+
+    @Override
```

Comment:
can rename method name to toString method-> the default method name for printing.